html/network.html

```diff
@@ -74,7 +74,7 @@ <h5 class="card-title"><strong>Output</strong></h5>
               .then(function(response) {
                 response.json().then(function(json) {
                   checkBox.checked = json["state"];
-                  var message = document.getElementById("resultMessage")
+                  var message = $('resultMessage');
```

Comment:
I think this needs to be `#resultMessage` to tell jQuery you're looking up an element by id.